testing/ios/IosUnitTests/IosUnitTests.xcodeproj/project.pbxproj

```diff
@@ -113,6 +115,15 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		0D1CE5D62334309900E5D880 /* Source-Common */ = {
+			isa = PBXGroup;
+			children = (
+				0D1CE5D7233430F400E5D880 /* FlutterChannelsTest.m */,
+			);
+			name = "Source-Common";
+			path = ../../../../shell/platform/darwin/common/framework/Source;
```

Comment:
Should the path match the group?  Meaning, should those unit tests be moved out of the same directory as the framework code?  I'm not familiar with the directory patterns here, but this path looks wonky.  These unit tests are just interleaved in the same directory as the real code?

Reply:
Yep, that's how it is done elsewhere in the engine.  It seems arbitrary for me.  I'm just trying to be consistent.